DataFormats/HcalDigi/src/classes_def.xml

```diff
@@ -108,4 +108,8 @@
     <version ClassVersion="10" checksum="2447500554"/>
    </class>
    <class name="edm::Wrapper<HcalLaserDigi>" splitLevel="0"/>
+   <class name="HcalUMNioDigi" ClassVersion="0">
```

Comment:
IIRC, 0 is for unversioned classes.
Are you sure about ``` ClassVersion="0" ```?
Normal classes should start from 3 (or 2, still possible)